tools/pipelines/templates/include-publish-npm-package-steps.yml

```diff
@@ -14,6 +14,9 @@ parameters:
   type: string
 
 - name: official
+  type: boolean
```

Comment:
Should we get rid of this parameter and now check for `eq(parameters.feedKind, 'official')`?

Reply:
I'm fine doing it afterwards.